### PR TITLE
Release/0.2.0

### PR DESCRIPTION
**Pull request by williamrolando88**
implemented API communication

### CODE REVIEW

File: src/modules/refreshScores.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import appendElements from './appendElements.js';
+
+export default async function refresh(e) {
```
Comment (bimbolabuari) [inline src/modules/refreshScores.js:3]: Let's refactor this to an arrow function instead of the `function` keyword

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @williamrolando88 , 

You have done a good job so far 🚀  

## Highlights 🎉 

- Implement the "Refresh" button.
- Implement the form "Submit" button.
- Use `async` and `await` JavaScript features to consume the API.

There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

File: src/modules/refreshScores.js (hunk 1 above)
Comment (williamrolando88) [reply src/modules/refreshScores.js]: fixed

**Review by EricMbouwe — APPROVED** [review]
### Hi @williamrolando88, 

# Status: Approved ✔️
Excellent job on this project. Keep it up 👏
Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**